module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -278,7 +290,15 @@ public function getResults()
                 : '';
             if (strlen($query) >= $this->useQueryMinLength) {
                 foreach ($nameToDatabase as $name => $databaseInfo) {
-                    if (str_contains($this->normalizeQueryString($name), $query)) {
+                    $normalizedName = $this->normalizeQueryString($name);
+                    if (str_contains($normalizedName, $query)) {
+                        $databases[$databaseInfo['url']] = $databaseInfo;
```

Comment:
I realize that both `if` blocks have the same body.  I think it's cleaner to keep them separate (instead of joining with an ||) since they are semantically separate tests, and one is already multi-line.

Reply:
What if you assign the conditions to variables, and then have one if statement ORing the variables together -- e.g.:

```
$nameContainsQuery = str_contains($normalizedName, $query);
$nameResemblesQuery = $this->useQueryMaxDifference &&
    levenshtein(substr($normalizedName, 0, strlen($query)), $query) <= $this->useQueryMaxDifference;
if ($nameContainsQuery || $nameResemblesQuery) {
    ...
```

(Those might not be the best names... and I don't insist on the change... but I think it might help with readability).